test/unit/spec/EnrollUser_spec.js

```diff
@@ -0,0 +1,157 @@
+/* eslint max-params: [2, 13], max-len: [2, 160] */
+define([
+  'okta',
+  '@okta/okta-auth-js/jquery',
+  'util/Util',
+  'helpers/mocks/Util',
+  'helpers/dom/EnrollUserForm',
+  'helpers/util/Expect',
+  'LoginRouter',
+  'sandbox',
+  'helpers/xhr/PROFILE_REQUIRED_UPDATE',
+  'helpers/xhr/PROFILE_REQUIRED_NEW',
+  'helpers/xhr/SUCCESS',
+  'helpers/xhr/UNAUTHENTICATED_IDX'
+],
+function (Okta, OktaAuth, LoginUtil, Util, EnrollUserForm, Expect, Router,
+  $sandbox, resProfileRequiredUpdate, resProfileRequiredNew, resSuccess, resUnauthenticatedIdx) {
+
+  var { _, $, Backbone } = Okta;
+  var itp = Expect.itp;
+
+  function setup (settings, resUnauthenticatedIdx) {
```

Comment:
I don't see a call with `settings` argument, so we can probably skip it
and there is only one call with `resUnauthenticatedIdx`. I think we should have 2 functions
```
function setupEnroll
function setupUnauthenticated()
```
or the other option is to refactor this one to something like
```
function setup (isUnauthenticated) {
  nextResponse = isUnathetnticated ? resUnauthenticatedIdx : resProfileRequiredUpdate;
```